kotlin-result-coroutines/build.gradle.kts

```diff
@@ -31,6 +31,7 @@ kotlin {
             dependencies {
                 implementation(kotlin("test-junit"))
```

Comment:
I think we can get rid of these now with https://blog.jetbrains.com/kotlin/2021/04/kotlin-1-5-0-rc-released/#test-library-changes

Might be worth making a separate PR though